org.koreader.KOReader.yml

```diff
@@ -13,49 +13,32 @@ finish-args:
   - --talk-name=org.freedesktop.Notifications
   - --filesystem=host
 modules:
-  - name: koreader
+  - shared-modules/SDL2/SDL2-2.26.1-with-libdecor.json
+  - shared-modules/lua5.1/lua-5.1.5.json
```

Comment:
Hasn't it used luajit before?

Reply:
Does `./kodev release linux` do it? That's what is unclear to me.

Reply:
So `lua` can be removed?